From 8eab26ae56b087fca043f59f7b3aa3cfca12cc05 Mon Sep 17 00:00:00 2001
From: John Zhuge <jzhuge@cloudera.com>
Date: Fri, 26 May 2017 10:25:28 -0700
Subject: [PATCH 2471/2540] HADOOP-14100. Upgrade Jsch jar to latest version
 to fix vulnerability in old versions. Contributed
 by Vinayakumar B and Brahma Reddy Battula.

(cherry picked from commit 2875462c2b21de0ab86fc000c0241dcf608d4450)

Change-Id: I55732609754086edf0d624271812fcd59ba43b8a
(cherry picked from commit dae88beca919ec6590d2bf0d19864534cb76a39c)
(cherry picked from commit acb1a8a3b99cbee85b84b87ef9261ccf95c25b5e)
---
 hadoop-project/pom.xml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-project/pom.xml b/hadoop-project/pom.xml
index d7cb433..4cf0120 100644
--- a/hadoop-project/pom.xml
+++ b/hadoop-project/pom.xml
@@ -819,7 +819,7 @@
       <dependency>
         <groupId>com.jcraft</groupId>
         <artifactId>jsch</artifactId>
-        <version>0.1.42</version>
+        <version>0.1.54</version>
       </dependency>
       <dependency>
         <groupId>org.apache.htrace</groupId>
-- 
1.7.9.5

